BaseApp/COD50006.txt

```diff
@@ -0,0 +1,126 @@
+OBJECT Codeunit 50006 DotNet_GzipStream
+{
+  OBJECT-PROPERTIES
+  {
+    Date=18-07-10;
+    Time=21:23:58;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      DotNetGZipStream@17024400 : DotNet "'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.Compression.GZipStream";
+      TextErrorReadNotSupported@17024401 : TextConst 'ENU=Stream cannot be read from';
+      TextErrorWriteNotSupported@17024402 : TextConst 'ENU=Stream cannot be written to';
+
+    [External]
+    PROCEDURE InitGzipStreamFromTempBlob@17024400(VAR TempBlob@1000 : Record 99008535;CompressMode@17024401 : Boolean);

```

Comment:
Assuming you have read my comments in #82, you know what a helper function is and where it should go. Please isolate helpers in separate codeunits.